pytorch/buildspec-graviton-1-10.yml

```diff
@@ -0,0 +1,39 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+framework: &FRAMEWORK pytorch
+version: &VERSION 1.10.0
+os_version: &OS_VERSION ubuntu20.04
+short_version: &SHORT_VERSION "1.10"
+arch_type: graviton
+
+repository_info:
+  inference_repository: &INFERENCE_REPOSITORY
+      image_type: &INFERENCE_IMAGE_TYPE inference
+      root: !join [ *FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
+      repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *INFERENCE_IMAGE_TYPE, "-", graviton]
+      repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
+
+context:
+  inference_context: &INFERENCE_CONTEXT
+    torchserve-e3-entrypoint:
+      source: docker/build_artifacts/torchserve-e3-entrypoint.py
+      target: torchserve-e3-entrypoint.py
+    deep_learning_container:
+      source: ../../src/deep_learning_container.py
+      target: deep_learning_container.py
+    config:
+      source: docker/build_artifacts/config.properties
+      target: config.properties
+
+images:
+  BuildE3PytorchGravitonCPUInferencePy3DockerImage:
+    <<: *INFERENCE_REPOSITORY
+    build: &PYTORCH_CPU_INFERENCE_PY3 false
+    image_size_baseline: 10000
+    device_type: &DEVICE_TYPE cpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py38
+    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION , "-e3"]
+    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.graviton.e3., *DEVICE_TYPE ]
```

Comment:
dockerfile needs to be renamed to .ec2.

Reply:
Noticing the yml is 1.10 and not 1.12. Also this is graviton 1.12. Would recommending renaming dockerfile to Dockerfile.graviton.cpu It will be supporting EC2 and sagemaker in the end.

Reply:
Made couple of changes to the buildspec-graviton-1.10 file to accommodate the e3 to ec2 change!